style.css

```diff
@@ -446,3 +497,312 @@ input[type="submit"] {
   background: #0d151e;
   border-radius: 10px;
 }
+
+@media screen and (min-width: 768px) {
+  #header {
+    justify-content: space-around;
+  }
+
+  .model_1-description p {
+    font-size: 20px;
+    line-height: 28px;
+    width: 80%;
+    margin-bottom: 0;
+  }
+
+  .model-2-description p {
+    width: 90%;
+    font-size: 17px;
+    line-height: 24px;
+    margin: 0;
+  }
+
+  .description_contact p {
+    width: 70%;
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  #owner_name p {
+    font-size: 20px;
+    line-height: 20px;
+  }
+
+  #hamburger-button {
+    display: none;
+  }
+
+  #myDropdown {
+    position: static;
+    display: flex;
+    z-index: 0;
+    align-items: center;
+    justify-content: center;
+  }
+
+  #headline {
+    height: 100vh;
+    flex-direction: row-reverse;
+    justify-content: flex-end;
+    align-items: center;
+    gap: 2rem;
+    background-image: url("./images/Illustration-header.png");
+    background-repeat: no-repeat;
+    background-size: 50%;
+    background-position: center right;
+  }
+
+  #introduction {
+    width: 50%;
+  }
+
+  .model_1-title h1 {
+    font-size: 40px;
+    line-height: 52px;
+    text-align: left;
+    margin-bottom: 0;
+  }
+
+  .model_2-title h1,
+  .item-2-title h1 {
+    width: 90%;
+    font-family: "Crete Round", sans-serif;
+    font-style: normal;
+    font-weight: 300;
+    font-size: 32px;
+    line-height: 44px;
+    color: #fff;
+    margin: 0;
+  }
+
+  .item-3-title h1 {
+    width: 90%;
+    font-family: "Crete Round", sans-serif;
+    font-style: normal;
+    font-weight: 300;
+    font-size: 32px;
+    line-height: 44px;
+    color: #fff;
+    margin: 0;
+  }
+
+  .item-2 .model_2-title h1 {
+    display: none;
+  }
+
+  .item-2-title,
+  .model_2-title {
+    display: flex;
+    justify-content: center;
+  }
+
+  .item-3 .model_2-title h1 {
+    display: none;
+  }
+
+  #introduction h1 {
+    font-size: 72px;
+    line-height: 82px;
+  }
+
+  #Occupation {
+    font-size: 32px;
+    line-height: 44px;
+  }
+
+  #qualification {
+    font-size: 20px;
+    line-height: 32px;
+  }
+
+  .contact-link {
+    height: max-content;
+    width: fit-content;
+    margin: auto 0;
+  }
+
+  #contact_infos {
+    flex-direction: column;
+    align-items: stretch;
+    justify-content: space-around;
+  }
+
+  #works_title {
+    width: 90%;
+    margin: 0 auto;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  #works_title h1 {
+    width: 75%;
+    margin-right: 0;
+  }
+
+  #work_separator {
+    width: 100%;
+    height: 0;
+  }
+
+  #card_works {
+    grid-template-columns: auto auto auto;
+    grid-column-gap: 5rem;
+    grid-row-gap: 10rem;
+    padding: 0;
+    width: 80%;
+    margin: 0 auto;
+  }
```

Comment:
- [ ] I think you may need to remove the row-gap between the card works because I noticed there is nothing of such in the design template 😄 .